src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +141,39 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
```

Comment:
Key question may be: should the `meta` be part of the signed request? I don't see a problem myself tbh, and ofc we need to wait til LP supports it, but did you have any thoughts about it?

Reply:
tl;dr: no.

I don't think it should be. AFAIK, the signature code already picks and chooses the attributes it signs, and ignores any others.

I can't think of any advantage to signing the `meta` (which, as its name indicates, is not part of the `request`), and I'd be worried about messing up the signature code.

Reply:
I was wrong.